src/index/coinstatsindex.cpp

```diff
@@ -402,24 +409,24 @@ bool CoinStatsIndex::ReverseBlock(const CBlock& block, const CBlockIndex* pindex
     for (size_t i = 0; i < block.vtx.size(); ++i) {
         const auto& tx{block.vtx.at(i)};
 
-        for (size_t j = 0; j < tx->vout.size(); ++j) {
+        for (uint32_t j = 0; j < tx->vout.size(); ++j) {
```

Comment:
Doesn't `vector<…>::size()` return a `size_t`, making this effectively a cast?
(apparently not necessarily: which `size_type` it returns is C++ library dependent)